Automation Examples/010.md

```diff
@@ -0,0 +1,200 @@
+# Process
+
+Open http://nseindia.com/corporates/content/eq_research_reports_listed.htm and for the first 50 companies, download a recent month’s report. Save it in a separate folder **Today’s date/Company Name/PDF Name.pdf**
+
+# Steps to automate
+
+#### 1. Create variables - `url` storing the link given and `basePath` containing the path to the main folder
+
+```G1ANT
+♥url = http://nseindia.com/corporates/content/eq_research_reports_listed.htm
+♥basePath = C:\Users\dell\Desktop\PDF Automation
+```
+
+#### 2. Create a main procedure in which you open the website and call it after the initialised variables section
+
+```G1ANT
+call DownloadPdfFiles
+
+procedure DownloadPdfFiles
+    selenium.open chrome url ♥url
+end procedure
```

Comment:
OpenSeleniumOnDownloadPage or somethings. This procedure does not download pdf files